.github/workflows/build-sample.yml

```diff
@@ -9,6 +9,9 @@ on:
       path:
         required: true
         type: string
+      module:
+        default: "app"
```

Comment:
is the default "app"? or "mobile"?

Reply:
The default should be "app" here since `build-sample.yml` is a shared workflow across all of our samples which all use "app" for the app module with the exception of Jetcaster